<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopAppClass</class>
 <widget class="QWidget" name="DesktopAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>911</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wukong</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1191</width>
     <height>881</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="loginTab">
    <attribute name="title">
     <string>Login</string>
    </attribute>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>229</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>299</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>380</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>482</x>
       <y>229</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>300</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>28</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 28pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Wukong</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="patientListTab">
    <attribute name="title">
     <string>Patient list</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>1100</width>
       <height>631</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>1101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Patient List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createNewPatientButton">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>750</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Create New Patient</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>750</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="patientListPreviousPageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Previous page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patientListPageLabel">
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Page 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="patientListNextPageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Next page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="patientTab">
    <attribute name="title">
     <string>Patient</string>
    </attribute>
    <widget class="QTableView" name="tableViewPatient">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>1100</width>
       <height>631</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="captureNewButton">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>750</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Capture New</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>1101</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="patientName">
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Name: Jerry Tam</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patientAge">
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Age: XX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patientSubjectNumber">
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Subject Number: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="captureTab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Capture</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsViewVideo4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>631</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar {height:0px; width:0px;}</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewImage">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>410</y>
       <width>591</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar {height:0px; width:0px;}</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>730</y>
       <width>321</width>
       <height>39</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton2">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton3">
        <property name="text">
         <string>Color to Depth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton4">
        <property name="text">
         <string>Depth to Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewVideo5">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>70</y>
       <width>381</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar {height:0px; width:0px;}</string>
     </property>
    </widget>
    <widget class="QLabel" name="saveInfoCaptureTab">
     <property name="geometry">
      <rect>
       <x>466</x>
       <y>790</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="recordingIndicatorText">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 5);
</string>
     </property>
     <property name="text">
      <string>Recording</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showInExplorer">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>820</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show In Explorer</string>
     </property>
    </widget>
    <widget class="QLabel" name="recordingGif">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>36</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="recordingElapsedTime">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="text">
      <string>00:25</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>760</y>
       <width>442</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="captureButton">
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButtonCaptureTab">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveVideoButton">
        <property name="text">
         <string>Start Recording</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="annotateButtonCaptureTab">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="annotateTab">
    <attribute name="title">
     <string>Analysis</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsViewAnnotation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>512</width>
       <height>288</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar {height:0px; width:0px;}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="annotationsText">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>480</y>
       <width>271</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>790</y>
       <width>505</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="saveButtonAnnotateTab">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="annotateButtonAnnotateTab">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Fetch annotations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reportButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Generate report</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewAnnotation2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>30</y>
       <width>512</width>
       <height>288</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar {height:0px; width:0px;}</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewImageUrl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar {height:0px; width:0px;}</string>
     </property>
    </widget>
    <widget class="QLabel" name="saveInfoAnnotateTab">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>460</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmLandmarksButton">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>350</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="alignmentTab">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Alignment</string>
    </attribute>
    <widget class="QPushButton" name="loadColorButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>200</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload RGB video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadDepthButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>240</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Depth video</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorFilename">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="depthFilename">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>240</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="alignButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>300</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Align RGBD video</string>
     </property>
    </widget>
    <widget class="QLabel" name="messageText">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="desktopapp.qrc"/>
 </resources>
 <connections/>
</ui>
